pokecli.py

```diff
@@ -201,19 +208,27 @@ def main():
     logger.log('[x] PokemonGO Bot v1.0', 'green')
     logger.log('[x] Configuration initialized', 'yellow')
 
-    try:
-        bot = PokemonGoBot(config)
-        bot.start()
+    finished = False
 
-        logger.log('[x] Starting PokemonGo Bot....', 'green')
+    while not finished:
+        try:
+            bot = PokemonGoBot(config)
+            bot.start()
 
-        while True:
-            bot.take_step()
+            logger.log('[x] Starting PokemonGo Bot....', 'green')
 
-    except KeyboardInterrupt:
-        logger.log('[x] Exiting PokemonGo Bot', 'red')
-        # TODO Add number of pokemon catched, pokestops visited, highest CP
-        # pokemon catched, etc.
+            while True:
+                bot.take_step()
+
+        except KeyboardInterrupt:
+            logger.log('[x] Exiting PokemonGo Bot', 'red')
+            finished = True
+            # TODO Add number of pokemon catched, pokestops visited, highest CP
+            # pokemon catched, etc.
+
+        except NotLoggedInException:
+            logger.log('[x] Error while connecting to the server, please wait %s minutes' % config.reconnecting_timeout, 'red')
+            time.sleep(config.reconnecting_timeout)
```

Comment:
time.sleep(X) sleeps X seconds.
but it's supposed to wait X minutes here, right?